### ISSUE TITLE
Unable to fetch "CEN/CENELEC Guide 6"

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-cen#8]
```sh
$ bundle exec relaton fetch "CEN/CENELEC Guide 6"
[relaton] (CEN/CENELEC Guide 6) not found.
No matching bibliographic entry found
...
$ bundle exec relaton fetch "CEN-CLC Guide 6"
[relaton-cen] ("CEN-CLC Guide 6") fetching...
[relaton-cen] WARNING: no match found online for CEN-CLC Guide 6. The code must be exactly like it is on the standards website.
No matching bibliographic entry found
...
$ bundle exec relaton fetch "CEN/CLC Guide 6"
[relaton-cen] ("CEN/CLC Guide 6") fetching...
[isoics] code   not found in ICS list
[relaton-cen] ("CEN/CLC Guide 6") found CEN/CLC Guide 6:2014
<bibdata type="standard" schema-version="v1.2.3">
  <fetched>2023-08-24</fetched>
...
```

Only `CEN/CLC Guide 6` worked.

https://www.cencenelec.eu/media/Guides/CEN-CLC/cenclcguide6.pdf

**Comment by andrew2net** [comment]
Because the search [page](https://standards.cencenelec.eu/dyn/www/f?p=205:105:939759890551601:::::) returns only CEN/CLC

<img width="1369" alt="image" src="https://github.com/relaton/relaton-cen/assets/5623890/3020aea9-3188-4c3b-a8f1-14e9a16da6d5">

To be able to use different reference formats, we should integrate the pubid-cen gem into relaton-cen. The pubid-cen can parse "CEN-CLC Guide 6" and "CEN/CLC Guide 6" references, but it can't parse "CEN/CENELEC Guide 6". So, if we need to support the "CEN/CENELEC Guide 6" like references, we need to make the pubid-cen parse it.

**Comment by ronaldtse** [comment]
@andrew2net in this case, can you update the README / documentation that we should do "CEN/CLC" not "CEN/CENELEC" or "CEN-CLC"?

Once that's done we can close this. Thanks!

**Comment by ronaldtse** [comment]
Actually there is a question here. When we cite "CEN/CLC Guide 6", shouldn't we return "CEN/CLC Guide 6" instead of "CEN/CLC Guide 6:2014"?

**Comment by andrew2net** [comment]
@ronaldtse The "CEN/CLC Guide 6:2014" is the latest edition. So if a year isn't provided relaton-cen returns the latest edition. Do you mean we should create "CEN/CLC Guide 6" document with all editions as relations?

**Comment by ronaldtse** [comment]
If an addition year is not given than it is an undated reference. Isn't this how it works for ISO and IEC?

**Comment by andrew2net** [comment]
> Isn't this how it works for ISO and IEC?

No, they use a method `RelatonBib::BibliographicItem#to_most_recent_reference` which creates a document without a date and adds `instanceOf` relation with the found latest document. They do not create relations for all editions. Should we use the method in this gem?

**Comment by ronaldtse** [comment]
@andrew2net for all ISO-like bibliographic items, such as CEN, BSI, JIS, we need to allow undated references. Let's apply this.

**Comment by andrew2net** [comment]
fixed in v1.16.2
```sh
relaton fetch "CEN/CLC Guide 6"
[relaton-cen] (CEN/CLC Guide 6) fetching...
[isoics] code   not found in ICS list
[relaton-cen] (CEN/CLC Guide 6) found `CEN/CLC Guide 6`
<bibdata type="standard" schema-version="v1.2.4">
  ...
```
